src/main/features/response/views/evidence.njk

```diff
@@ -0,0 +1,82 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, linkButton, saveAndContinueButton, errorSummary %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('List your evidence') %}
+
+{% macro evidenceRow(index, form) %}
+
+  {% set typeValue = form.valueFor('rows[' + index + '][type]') %}
+  {% set descriptionValue = form.valueFor('rows[' + index + '][description]') %}
+
+  {% set errorDescription = form.errorFor('rows[' + index + '][description]').length > 0 %}
+
+  <div class="timeline-row {% if errorDescription %} form-group-error {% endif %}">
+    <label class="visually-hidden" for="rows[{{ index }}][type]">{{ index + 1 }}{{ t('. Evidence') }}</label>
+    <select class="form-control-select" name="rows[{{ index }}][type]" id="rows[{{ index }}][type]">
+      <option>-- Please make a selection --</option>
+      <option value="CONTRACTS_AND_AGREEMENTS" {% if typeValue == 'CONTRACTS_AND_AGREEMENTS' %}selected{% endif %}>Contracts and agreements</option>
```

Comment:
I think we can build this with a for loop and get the values from the .all() function on the type?